llvm/test/CodeGen/AMDGPU/structurizer-keep-perf-md.ll

```diff
@@ -0,0 +1,72 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -mtriple=amdgcn-- -S -structurizecfg %s | FileCheck -check-prefix=OPT %s
```

Comment:
```suggestion
; RUN: opt -S -mtriple=amdgcn-- -passes=structurizecfg %s | FileCheck -check-prefix=OPT %s
```

This belongs in test/Transform/StructurizeCFG, and should use new PM

Reply:
Moved into `Transforms/StructurizeCFG/structurizer-keep-perf-md.ll` and updated to match